dts/bindings/serial/uart-controller.yaml

```diff
@@ -42,3 +42,13 @@ properties:
       - 7
       - 8
       - 9
+  tx-invert:
+    type: boolean
+    description: |
+      Invert the binary logic of tx pin. When enabled, physical logic levels are inverted and
+      we use 1=Low, 0=High instead of 1=High, 0=Low.
+  rx-invert:
+    type: boolean
+    description: |
+      Invert the binary logic of rx pin. When enabled, physical logic levels are inverted and
+      we use 1=Low, 0=High instead of 1=High, 0=Low.
```

Comment:
These properties cannot be made global for all UARTs, as not all UARTs support these. Having them available for all UARTs will give users the impression that setting these properties will result in inversion, which may or may not be true.

One option could be to define them here, but only `include` them in the bindings, where they are actually supported. Take a look at the pinctrl bindings for examples.

Reply:
I too thought that but found no reference of conditional include for dts properties. 
I will check pinctrl and get back. 
Thanks!

Reply:
I think it is possible to define these in another binding yaml file. For those controller that supports these, they can include this new file in addition to `uart-controller.yaml`.

Reply:
resolved